src/Actions/BcgovSettings.php

```diff
@@ -59,11 +59,27 @@ public function bcgov_settings_page() {
 			}
 		}
 
+        // Check if the documentation url nonce field is set in the $_POST data.
+		if ( isset( $_POST['bcgov_documentation_url_nonce'] ) ) {
+
+			if ( wp_verify_nonce( $doc_nonce, 'bcgov_documentation_url_nonce' ) ) {
+				if ( isset( $_POST['bcgov_documentation_url'] ) ) {
+					$documentation_url = sanitize_text_field( $_POST['bcgov_documentation_url'] );
+					update_option( 'bcgov_documentation_url_settings', $documentation_url );
+				}
+			}
+		}
+
         // Get the current Google Site Name setting.
         $google_site_name = get_option( 'bcgov_google_site_name_settings', '' );
 
         $nonce = wp_create_nonce( 'bcgov_google_site_name_nonce' );
 
+        // Get the current Site Documentation URL setting.
+        $documentation_url = get_option( 'bcgov_documentation_url_settings', '' );
+
+        $doc_nonce = wp_create_nonce( 'bcgov_documentation_url_nonce' );
```

Comment:
is this not the same form as the google nonce, wondering why we have so many nonces, should only need one

Reply:
It wasn't clear to me that you only need one nonce per form.